var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -33,8 +33,7 @@ class Nwchem(Package):
     homepage = "http://www.nwchem-sw.org"
     url      = "http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz"
 
-    version('6.6', 'c581001c004ea5e5dfacb783385825e3',
-            url='http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz')
+    version('6.6.revision27746-src.2015-10-20', 'c581001c004ea5e5dfacb783385825e3')
```

Comment:
> One of my goals here was that a url_for_version or version-specific url wouldn't be necessary anymore. 

let's keep them for packages which don't call tarbals _right_. 

from  http://www.nwchem-sw.org/index.php/Main_Page 

> NWChem 6.6 is available for download